source/client/components/CVDocument.ts

```diff
@@ -183,8 +183,8 @@ export default class CVDocument extends CRenderGraph
 
         if (!mergeParent) {
             this.clearNodeTree();
-            this.ins.title.setValue(null);
         }
+        this.ins.title.setValue(null);
```

Comment:
This works for updating the title but certainly seems like it could have unintended side effects. Maybe I should check if the new document has a title before clearing this value?

Reply:
Of course, one alternative might be to leave this as it originally was and modify the call from `evaluateProps` to not merge the parent. The issue I have there is that the entire viewer disappears, presumably because of `clearNodeTree()` here.

Reply:
With a little more time, I've tracked this down to where the update should actually happen in `CVDocument`. This alone is also enough to force the title to update, but is likewise prone to unintended side effects:
```diff
# CVDocument.ts, line 293
-        if (event.add && !propTitle.value) {
+        if (event.add) {
             meta.once("load", () => {
                 this.titles = meta.collection.get("titles") || {};
```

I can see a number of ways around this at this point, but I'm not sure what the safest solution is. I'm inclined to unset `title` in `ExplorerApplication.evaluateProps()`. This would seem to localize the effects to only Explorer where changing the document property is (perhaps) expected to change the scene:
```diff
# ExplorerApplication.ts, line 325
         if (props.document) {
             // first loading priority: document
             props.document = props.root ? props.document : manager.getAssetName(props.document);
+            this.documentProvider.activeComponent.ins.title.setValue(null);
             this.loadDocument(props.document, undefined, props.quality)
```
I'm going to push this approach for now. If there's a better way, let me know what you prefer.